src/routes/api/tripRequest.route.js

```diff
@@ -0,0 +1,10 @@
+import express from "express";
+import tokenVerify from "../../middlewares/tokenAuthentication"
+import tripRequestController from "../../controllers/trip.request";
+
+const router = express.Router();
+const { sendTripRequest } = tripRequestController;
+
+router.post('/tripRequest/:userToken', tokenVerify, sendTripRequest);
```

Comment:
why are using user token?

Reply:
I used the user token to get the `user id` from it.